Gemfile

```diff
@@ -2,15 +2,19 @@ source 'https://rubygems.org'
 
 gemspec
 gem 'fog-aws'
-gem 'capybara', '~> 2.3.0'
-gem 'database_cleaner', '~> 0.9.1'
-gem 'rspec-rails', '~> 3.2.0'
-gem 'poltergeist'
 gem 'ammeter'
-gem 'test-unit', platforms: [:ruby_22]
 
-group :development do
+group :development, :test do
```

Comment:
But that's my point - what other environments are there? There is no environment where you don't want the test gems, is there?